addons/main/functions/Profiles/fnc_doesProfileAllow.sqf

```diff
@@ -0,0 +1,41 @@
+#include "script_component.hpp"
+/*
+ * Author: joko // Jonas
+ * Group profiles handler -- Checks and runs AI profiles -- group actions with the fnc_profileXxx prefix.
+ *
+ * Arguments:
+ * 0: Group/Unit <OBJECT, GROUP>
+ * 1: Profile Tactic <STRING>
+ *
+ * Return Value:
+ * Tactic is allowed to Execute
+ *
+ * Example:
+ * [bob] call lambs_danger_fnc_doesProfileAllow;
+ *
+ * Public: No
+*/
+params [["_target", objNull, [objNull, grpNull]], ["_tactic", "unkown", [""]]];
+
+private _profile = _target getVariable [QGVAR(AIProfile), "default"];
+
+private _profileData = GVAR(ProfilesNamespace) getVariable toLower(_profile);
+if (isNil "_profileData") then {
+    _profileData = GVAR(ProfilesNamespace) getVariable "Default";
+};
+_tactic = toLower(_tactic);
+private _value = [_profileData, _tactic] call CBA_fnc_hashGet;
+if (_value isEqualType {}) then {
```

Comment:
I feel like a nil check is needed here for `_value` in the case of some tactic is being requested that is not configured.

Reply:
that is already handled by CBA. in the creation of the hash you put in a default return type when it is not defined current Default return is true. ref: https://github.com/nk3nny/LambsDanger/pull/164/files#diff-2f2cb255b4a09218514aa7e856030d9cR23